### DISCUSSION TITLE
where I can have access to the server logs?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2363] [Help] [answered]
Hello,

I have a request that produces an error and I would like to understand why. So I am looking for the server logs...

Do you know where they are?

Thank you

**Comment by shogunpurple** [comment, marked as answer]
hi @oswaldB - you can do this through the docker CLI:

`docker ps` will show you all your containers and their container ID. There are 2 main services that run budibase, the worker and the app server. The error you are seeing is probably in the app server.

![Screenshot 2021-08-16 at 09 08 52](https://user-images.githubusercontent.com/11256663/129532014-7c9bc9f2-72ad-4d2f-a4d8-43e58c64adaa.png)

run `docker logs <container-id>` to get all the logs. You can also do `docker logs <container-id> --follow` if you want to stream them.

Hope this helps!

**Reply by oswaldB** [reply]
thank you. I can have access to them!

Unfortunatly, I cannot find the information I am looking for. 

Thank you I will keep going on.

Have a nice day
